tests/unit/common/test_test.py

```diff
@@ -1,191 +0,0 @@
-from __future__ import (
```

Comment:
The contents of this file were merged into `tests/unit/test/test_stub_service.py` (below).